index.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'coordinate-parser' {
+    class CoordinateParser {
+        constructor(coordinateString: string)
+        getLatitude(): number
+        getLongitude(): number
+        isValidPosition(): boolean
```

Comment:
There is no method with this name in the source code.

Reply:
You are right. Removed.